package.json

```diff
@@ -54,17 +54,17 @@
   "devDependencies": {
     "@skyleague/event-horizon": "./",
     "@skyleague/event-horizon-dev": "^4.0.11",
-    "@skyleague/node-standards": "^3.10.5",
-    "@skyleague/therefore": "^3.0.2",
-    "@types/node": "^20.8.9",
+    "@skyleague/node-standards": "^4.0.0",
+    "@skyleague/therefore": "^3.1.0",
+    "@types/node": "^20.10.5",
     "aws-sdk-client-mock": "^3.0.0",
     "aws-sdk-client-mock-jest": "^3.0.0",
     "got": "^13.0.0",
```

Comment:
v14 is OK now

Reply:
we need to update this one in `therefore` first, I'm getting conflicting dependency resolution

Reply:
updated!